Compile error on Ubuntu 18.04

Hello folks,

When trying to compile on Ubuntu 18.04, I got the following compile error with GCC 7.4.0:
```
modules/voxel/meshers/dmc/voxel_mesher_dmc.cpp: In member function 'virtual void VoxelMesherDMC::build(VoxelMesher::Output&, const VoxelBuffer&, int)':
modules/voxel/meshers/dmc/voxel_mesher_dmc.cpp:1469:15: error: no match for 'operator=' (operand types are 'VoxelMesherDMC::Stats' and '<brace-enclosed initializer list>')
  _stats = { 0 };
```

After changing this line of code on **/modules/voxel/meshers/dmc/voxel_mesher_dmc.cpp(1469)**:
```
_stats = { 0 };
```
To:
```
_stats = {};

```
It worked fine.

--- a/meshers/dmc/voxel_mesher_dmc.cpp
+++ b/meshers/dmc/voxel_mesher_dmc.cpp
@@ -1466,7 +1466,7 @@ void VoxelMesherDMC::build(VoxelMesher::Output &output, const VoxelBuffer &voxel
 	// - Voxel data must be padded
 	// - The non-padded area size is cubic and power of two
 
-	_stats = { 0 };
+	_stats = { };
 
 	if (voxels.is_uniform(VoxelBuffer::CHANNEL_ISOLEVEL)) {
 		// That won't produce any polygon

